src/constants/notifications.ts

```diff
@@ -250,3 +258,107 @@ export type NotificationPreferences = {
 } & { version: string };
 
 export const DEFAULT_TOAST_AUTO_CLOSE_MS = 5000;
+
+export const TOP_100_WALLET_ADDRESSES = [
```

Comment:
Is there a way we could get this from a backend? We shouldn't have hardcoded addresses

Reply:
we can also have dos input these and use statsig dynamic configs, from what i've gathered we don't seem concerned about this but let me confirm